rust/agama-server/src/network/nm/error.rs

```diff
@@ -19,7 +19,10 @@
 // find current contact information at www.suse.com.
 
 //! NetworkManager error types
+use std::net::AddrParseError;
+
```

Comment:
is there reason for this empty line? I do not get why the first one is separated from rest.